### PR TITLE
List Greencoin (GRN)

### PR DESCRIPTION
**Pull request by CryptorClub**
Site: http://greencoin.life
Web wallet: https://wallet.greencoin.life
Official pool: https://greencoin.space
Explorer: http://explorer.greencoin.space
Source code: https://github.com/CryptorClub/GreenCoin

ANN: https://bitcointalk.org/index.php?topic=3103301

### CODE REVIEW

**Comment by blabno** [comment]
Please do not use Travis as your local test runner.
Squash those commits with proper message.

File: assets/src/main/java/bisq/asset/coins/Greencoin.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+package bisq.asset.coins;
+
+import bisq.asset.Coin;
+import bisq.asset.DefaultAddressValidator;
+
+public class Greencoin extends Coin {
+
+    public Greencoin() {
+        super("Greencoin", "GRN", new DefaultAddressValidator());
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Greencoin.java]: Do not use deprecated `DefaultAddressValidator`

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

**Comment by CryptorClub** [comment]
@blabno please give me advice, what to use instead of travis. I've squash commit into one

File: assets/src/main/java/bisq/asset/coins/Greencoin.java (hunk 1 above)
Comment (CryptorClub) [reply assets/src/main/java/bisq/asset/coins/Greencoin.java]: I've use Base58 validator

**Comment by CryptorClub** [comment]
> Please do not use Travis as your local test runner.
> Squash those commits with proper message.

How to use another test runner? It seems, that it was runned automatically, when I published commit...
I've squashed commits into one.

**Comment by blabno** [comment]
Apply Bisq code style and resolve conflicts.

**Comment by blabno** [comment]
Is this ICO https://icobench.com/ico/greencoin ?

**Comment by ManfredKarrer** [comment]
Greencoin is an ICO coin. Bisq does not list ICO coins.

**Comment by CryptorClub** [comment]
1. GreenCoin GRN is not GNC coin. And isn't ICO

**Pull request closed without merging**